conanfile.py

```diff
@@ -11,27 +11,25 @@ class InexorConan(ConanFile):
 
     requires = (
         "benchmark/1.5.0",
-        "glm/0.9.9.5",
-        "gtest/1.8.1",
-        "spdlog/1.4.2",
-
         "boost_property_tree/1.69.0@bincrafters/stable",
         "boost_signals2/1.69.0@bincrafters/stable",
         "boost_range/1.69.0@bincrafters/stable",
-        "freetype/2.9.1@bincrafters/stable",
-        "openal/1.19.0@bincrafters/stable",
-
-        "magic_enum/0.6.3@neargye/stable",
-
-        "jsonformoderncpp/3.7.0@vthiery/stable",
-
         "boost-di/1.1.0@inexorgame/stable",
         "boost-te/19.Jan.19@inexorgame/stable",
-        "cpp.react/legacy1@inexorgame/stable",
-
         "crossguid/06-03-19@inexorgame/testing",
+        "cppgraphqlgen/3.0.4@inexorgame/testing",
+        "cpp.react/legacy1@inexorgame/stable",
+        "freetype/2.9.1@bincrafters/stable",
```

Comment:
Please try if you can update to the CCI variant of freetype and version 2.10.1
```suggestion
        "freetype/2.10.1",
```